base/abstractarraymath.jl

```diff
@@ -200,31 +200,61 @@ function repmat(a::AbstractVector, m::Int)
     return b
 end
 
-# Generalized repmat
-function repeat{T}(A::AbstractArray{T};
-                   inner::Array{Int} = ones(Int, ndims(A)),
-                   outer::Array{Int} = ones(Int, ndims(A)))
+"""
+    repeat(A::AbstractArray; inner=(), outer=())
```

Comment:
Yeah, the old code used a similar "trick". The actual value `ones(Int, ndims(A))` is a bit long, though not the end of the world. I've also tried another solution in which passing an empty collection is supported and is internally replaced with a vector of ones. Which one would you prefer?